### PR TITLE
Refactor generate doc files

### PR DESCRIPTION
**Pull request by zeeshanu**

### CODE REVIEW

File: app/Helpers/HtmlToDocHelper.php [hunk 1]
```diff
@@ -4,285 +4,119 @@
...
 class HtmlToDocHelper
 {
-	var $docFile="";
-	var $title="";
-	var $htmlHead="";
-	var $htmlBody="";
+	public $docFile  = "";
+	public $title    = "";
+	public $htmlHead = "";
+	public $htmlBody = "";
```
Comment (kamranahmedse) [inline app/Helpers/HtmlToDocHelper.php]: protected if it is not going to be used outside

File: app/Helpers/HtmlToDocHelper.php [hunk 2]
```diff
@@ -4,285 +4,119 @@
...
 	
 	
 	/**
 	 * Constructor
 	 *
 	 * @return void
 	 */
-	function HTML_TO_DOC()
+	public function HTML_TO_DOC()
```
Comment (kamranahmedse) [inline app/Helpers/HtmlToDocHelper.php]: camel case function names `htmlToDoc()`

Comment (zeeshanu) [inline app/Helpers/HtmlToDocHelper.php]: :+1:

File: app/Helpers/HtmlToDocHelper.php (hunk 1 above)
Comment (zeeshanu) [inline app/Helpers/HtmlToDocHelper.php]: Good catch

**Pull request merged**